src/reducer/uptime.js

```diff
@@ -0,0 +1,106 @@
+import ActionTypes from '../constants/action-type';
+
+export const initialState =  {
+  isLoading: false
+};
+
+const uptime = (state=initialState, action)=> {
+
+  if (!action.type) {
+    console.log('Payload missing a type!', action);
```

Comment:
Why is this log statement used ?

Reply:
Right now I am just consoling the error if an action type is missing. I will probably show the respective error message to the user in this scenario.